base/test-file-will-remove

```diff
@@ -0,0 +1,4 @@
+testing
```

Comment:
remove this. it takes me very little time to see files that shouldn't be present in the files changed tab of the PR, or in a local git diff, so that means you should catch them yourself before requesting review.

Reply:
I need to be able to have something for my pipeline to detect? Is there another way to introduce a change within a directory to trigger the main functionality of the pipeline, which is to detect changes in a directory?

Reply:
Removed.

Reply:
that makes sense, but by the time it goes to review the file should be removed.